graphene-pk11/graphene-pk11-tests.ts

```diff
@@ -1,8 +1,6 @@
 /// <reference path="./graphene-pk11.d.ts" />
 
-import * as graphene from "graphene-pk11";
-
-// Example of Hashing from README.MD <https://github.com/PeculiarVentures/graphene#hashing>
+let graphene = require("graphene-pk11");
```

Comment:
Can you revert this back to `import *` 

and add trailing commas to `enum`s, that makes future merges easier.